### ISSUE TITLE
Fix losing user settings when getting new API key

### ISSUE THREAD

**Issue by mjftw** [mjftw/react-bitbucket-pull-request-board#18]
Now that we have live data we have a new issue.
After 2 hours the Bitbucket API key will have expired, and new requests will automatically cause a new token to be issued. Unfortunately this means sending the user back to the Bitbucket website briefly before getting redirected back to the PR board. This means that all session settings will be lost, causing all PRs to be removed from the board etc.

This could be remedied by using localstorage to store and restore session settings.
https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage

Another approach could be to avoid the redirect at all and use a Bitbucket API refresh token if the current token expired.

**Comment by mjftw** [comment]
This can be fixed by making some redux state persistent in local or session storage (once move to redux is done)
